.changeset/eleven-cougars-scream.md

```diff
@@ -0,0 +1,6 @@
+---
+"@digdir/designsystemet": patch
+"@digdir/designsystemet-theme": patch
+---
+
+Add support for `data-color` and `data-color-scheme` being defined on the same element
```

Comment:
```suggestion
Fix defined `data-color` and `data-color-scheme` on the same element using wrong color-scheme variables.
```

Reply:
Ok, but change the wording to fix as we agreed on in previous comments?

Reply:
Ok, it was not made clear you wanted the word "fix" in there.
Suggestion:
```suggestion
Fix `data-color` and `data-color-scheme` used on the same element not working in some browsers
```
It does not matter in the changelog why it did not work, only that it did not work in some browsers, and it is now fixed